### DISCUSSION TITLE
changing generated results of stored procedures and table value functions

### DISCUSSION THREAD

**Discussion by mariorancic01** [linq2db/linq2db#3764] [General]
I am using scaffolding tool with custom assembly. I want to change generated return type of stored procedures and table valued functions with known already generated entity. Is it possible,

Also is it possible to debug custom assembly.

**Comment by MaceWindu** [comment]
- currently it is not possible to change types for procedure/function records
- for debug I prefer to add `System.Diagnostics.Debugger.Launch()` call to debugged code

**Reply by mariorancic01** [reply]
thx for debugging help. it really is amazing. What a great tool you built. 
Now I see why we cannot change stored procedures and table value functions results , because with PreprocessEntity we can only preprocess tables and views.

**Reply by MaceWindu** [reply]
I've created feature request, but it will take some time to implement #3765 (no timeslot for at least next month)

**Reply by mariorancic01** [reply]
its ok, thx
